pkg/k8s/k8s_rolebinding.go

```diff
@@ -0,0 +1,60 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+// CreateRoleBinding creates a roleBinding
+func CreateRoleBinding(name, namespace string, labels map[string]string, role, serviceAccount string) error {
```

Comment:
thinking loudly, if we are going to use this function to create rolebinding role<->sa, we could add it to the comments to generate the docs correctly, to prevent any misunderstanding (rolebinding can bind role<->user for example), wdyt